src/components/onboarding/sign-up/SignUp.js

```diff
@@ -0,0 +1,256 @@
+import React, { Component } from 'react'
+import * as ReactRedux from 'react-redux'
+import PropTypes from 'prop-types'
+import { compose } from 'recompose'
+import { withStyles } from '@material-ui/core/styles'
```

Comment:
please remove dependency on the material design

Reply:
file deleted